README.md

```diff
@@ -0,0 +1,158 @@
+<div align="center" style="margin-bottom: 1em;">
+
+<img width=50% src="docs/assets/images/logo.png" alt="Fast-LLM Logo"></img>
+
+[![Tests][tests-badge]][tests]
+[![Docker Build][docker-badge]][docker]
+[![Documentation Build][docs-badge]][docs]
+[![License][license-badge]][license]
+
+*Accelerating your LLM training to full speed*
+
+Made with ❤️ by [ServiceNow Research][servicenow-research]
+
+</div>
+
+## Overview
+
+Fast-LLM is a new open-source library for training large language models. Its design focuses on speed, scalability, flexibility, and ease of use. Fast-LLM is built on top of [PyTorch][pytorch] and [Triton][triton] to provide a state-of-the-art training experience.
+
+## Why Fast-LLM?
+
+1. 🚀 **Fast-LLM is Blazingly Fast**:
+    - ⚡️ Optimized kernel efficiency and reduced overheads.
+    - 🔋 Optimized memory usage.
+    - ⏳ Low training time and cost.
+  
+2. 📈 **Fast-LLM is Highly Scalable**:
+    - 📡 Distributed training across multiple GPUs and nodes using 3D parallelism (Data, Tensor, and Pipeline).
+    - 🔄 Supports sequence length parallelism.
+    - 🧠 ZeRO-1, ZeRO-2, and ZeRO-3 offloading for memory efficiency.
+    - 🎛️ Support for mixed precision training.
+    - 🏋️‍♂️ Large batch training and gradient accumulation support.
+
+3. 🎨 **Fast-LLM is Incredibly Flexible**:
+    - 🤖 Compatible with all common language model architectures in a unified class.
+    - ⚡ Efficient dropless Mixture-of-Experts (MoE) support.
+    - 🧩 Customizable for language model architectures, data loaders, loss functions, and optimizers.
+    - 🤗 Seamless integration with [Hugging Face Transformers](https://huggingface.co/transformers/).
+
+4. 🎯 **Fast-LLM is Super Easy to Use**:
+    - 📦 Pre-built Docker images for quick deployment.
+    - 📝 Simple YAML configuration for hassle-free setup.
+    - 💻 Command-line interface for easy launches.
+    - 📊 Detailed logging and real-time monitoring features.
+    - 📚 Extensive documentation and practical tutorials.
```

Comment:
Really?

Reply:
Aspirationally, yes

Reply:
But we don't have it yet, so we need to add a "in progress" or something so we're not lying

Reply:
No need. I'll work on that this week. The docs are very outdated and incomplete and urgently need an overhaul.

Reply:
Even with a big overhaul it still won't qualify as "extensive"